client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -20,6 +18,7 @@
 		margin: 0;
 	}
 
+	.stepper-loader__progress-bar,
```

Comment:
It would be better if we don't entangle the framework with the steps. Can we pass a class name to the progress bar from the processing step then stylize against that class?

Reply:
Nice catch. I removed it because it was not used anymore!